rosidl_generator_py/resource/_msg_support.c.template

```diff
@@ -2,7 +2,6 @@
 
 #include <@(spec.base_type.pkg_name)/@(subfolder)/@(module_name)__struct.h>
 #include <@(spec.base_type.pkg_name)/@(subfolder)/@(module_name)__functions.h>
-
```

Comment:
Please don't remove these empty lines. The make the generated code more structured and therefore readable.